src/views/accounts/AccountChangeSecret/AccountChangeSecretList.vue

```diff
@@ -67,7 +67,7 @@ export default {
               can: vm.$hasPerm('accounts.view_changesecretexecution'),
               getRoute({ row }) {
                 return {
-                  name: 'AccountChangeSecretList',
+                  name: 'AccountChangeSecretDetail',
                   query: {
                     tab: 'AccountChangeSecretExecutionList',
                     automation_id: row.id
```

Comment:
No significant differences or issues found in the provided code snippet. In general, maintaining consistency across components is important to maintain readability and ease of maintenance. 

However, if you have specific concerns regarding performance, data flow, or documentation requirements, I would be glad to review it further with additional context!